book/01-introduction/sections/about-version-control.asc

```diff
@@ -30,7 +30,7 @@ RCS works by keeping patch sets (that is, the differences between files) in a sp
 (((version control,centralized)))
 The next major issue that people encounter is that they need to collaborate with developers on other systems.
 To deal with this problem, Centralized Version Control Systems (CVCSs) were developed.
-These systems, such as CVS, Subversion, and Perforce, have a single server that contains all the versioned files, and a number of clients that check out files from that central place. (((CVS)))(((Subversion)))(((Perforce)))
+These systems, such as CVS, Subversion and Perforce, have a single server that contains all the versioned files, and a number of clients that check out files from that central place. (((CVS)))(((Subversion)))(((Perforce)))
```

Comment:
In this case, it seems like the liberal sprinkling of commas is doing us all a disservice. What if we repunctuate to something like this:

> These systems (such as CVS, Subversion, and Perforce) have a single server that contains all the versioned files, and a number of clients that check out files from that central place.

That way it gets more readable AND I get to keep my Oxford comma. 😁

Reply:
I'm fine with whatever you decide. Tweak the pull request, I'm in the middle of more wide-sweeping changes. :-)

Reply:
Sadly I cannot push to your fork. :(

Reply:
OK, I'll resubmit with that change. (I thought I checked the box that lets you edit my pull requests, is that not what that means?)

Reply:
Ha, I was making that _way_ harder than I needed to. 😊